feat(changelog): generative-apis-deprecated-deprecation-llama-31-70b 2025-02-25 (#4491)
scaleway/docs-content: changelog/february2025/2025-02-25-generative-apis-deprecated-deprecation-llama-31-70b.mdx

---
title: Deprecation - Llama 3.1 70B
status: deprecated
author:

>     fullname: ''

Reviewer: no more need for slack channel (is no longer displayed on changelog)

    url: 'https://slack.scaleway.com'
date: 2025-02-25
category: ai-data
product: generative-apis
---

> Llama 3.1 70B is now deprecated, as the newer Llama 3.3 70B is available with similar or above performance in most use cases.
> Llama 3.1 70B will remain available through API until May 25th 2025, and redirected automatically to Llama 3.3 70B afterwards. Llama 3.1 8B is not affected by this change and remains in Active support.
> 

Reviewer 2: ```suggestion
Llama 3.1 70B is now deprecated. The new Llama 3.3 70B is available with similar or better performance in most use cases.
Llama 3.1 70B will remain available through the API until May 25th 2025. You will then be redirected automatically to the Llama 3.3 70B API afterwards. Llama 3.1 8B is not affected by this change and remains supported.
```
